luigi/tools/range.py

```diff
@@ -645,6 +647,76 @@ def infer_bulk_complete_from_fs(datetimes, datetime_to_task, datetime_to_re):
     return missing_datetimes
 
 
+class RangeMonthly(RangeBase):
+    """
+    Produces a contiguous completed range of a monthly recurring task.
+
+    Unlike the Range* classes with shorter intervals, this class does not perform bulk optimisation.
+    It is aassumed that the number of months is low enough not to motivate the increased complexity.
+    Hence, there is no class RangeMonthlyBase.
+    """
+    start = luigi.MonthParameter(
+        default=None,
+        description="beginning month, inclusive. Default: None - work backward forever (requires reverse=True)")
+    stop = luigi.MonthParameter(
+        default=None,
+        description="ending month, exclusive. Default: None - work forward forever")
+    months_back = luigi.IntParameter(
+        default=13,  # Little over a year
```

Comment:
what is the motivation for a default which is a `little over a year`?

Reply:
It is arbitrary, but it is common to desire to keep a window of datasets back to slightly more than an interesting interval, e.g. in order to be able to compare with last year's numbers. RangeDaily and RangeHourly have 100 days window back - slightly more than a quarter.